tests/Feature/Accessories/Api/AccessoryFilesTest.php

```diff
@@ -103,7 +103,7 @@ public function testAccessoryApiDownloadsFile()
         // Upload a file
         $this->actingAsForApi($user)
             ->post(
-                route('api.files.store', ['object_type' => 'accessories', 'id' => $accessory->id]), [
+                route('api.files.store', ['object_type' => 'accessories', 'id' => $accessory->id, 'order' => 'asc']), [
```

Comment:
This should be on line 137

Reply:
This is what happens when you’re throwing up code as you’re running out the door